### PR TITLE
[ticket/15115] Update CSS error checking Stylelintrc file

### PR DESCRIPTION
**Pull request by hanakin**
Checklist:

- [x] Correct branch: master for new features; 3.2.x, 3.1.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master / 3.2.x](https://area51.phpbb.com/docs/master/coding-guidelines.html), [3.1.x](https://area51.phpbb.com/docs/31x/coding-guidelines.html)
- [x] Commit follows commit message [format](https://wiki.phpbb.com/Git#Commit_Messages)

PHPBB3-15115

https://tracker.phpbb.com/browse/PHPBB3-15115

### CODE REVIEW

**Comment by hanakin** [comment]
!set Styles

**Comment by marc1706** [comment]
Sorry, fixing the merge conflict in the other PR I just merged caused a merge conflict here.

**Comment by hanakin** [comment]
no worries should be good once test complete

File: phpBB/styles/prosilver/theme/tweaks.css [hunk 1]
```diff
@@ -27,7 +28,7 @@ dd label input {
 	height: 20px\9;
 }
 
-/* IE8 often can't handle max-width in %, so we use px instead */
+/* IE8 often can"t handle max-width in %, so we use px instead */
```
Comment (marc1706) [inline phpBB/styles/prosilver/theme/tweaks.css]: Typo? ;)

Comment (hanakin) [inline phpBB/styles/prosilver/theme/tweaks.css]: to be fair this file needs deleted!

**Comment by hanakin** [comment]
closed to restart travis

**Comment by marc1706** [comment]
There was an issue with the line endings in your last commit. Fixed that for you.

**Comment by marc1706** [comment]
Merged via https://github.com/phpbb/phpbb/commit/f36d5691c8ca8e8d6f02f55d51b8d1de66d9cfef

**Pull request closed without merging**